### ISSUE TITLE
[Inline message] Update component in Figma - remove action buttons

### ISSUE THREAD

**Issue by anna-lach** [sl-design-system/components#2033]
### Provide a general summary of the issue here

According to the accessibility slack channel discussion: https://sanoma.slack.com/archives/CFRULMYLD/p1747122952057719,
we've removed `action slot` and examples with action buttons (we do not recommend adding interactive elements inside the inline message component, interactive elements are not properly announced by aria-live). 

Please check https://github.com/sl-design-system/components/issues/1024#issuecomment-2374050256.

Action buttons need to be removed from Figma component as well.

### 👤 Your name

Anna Sobczak

### 🧢 Your Product/Team

SLDS
